src/main/java/com/resource/management/services/controller/LastUpdatedTimestampService.java

```diff
@@ -0,0 +1,49 @@
+package com.resource.management.services.controller;
+
+import java.time.Instant;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.resource.management.api.services.ServicesListUpdatedNotification;
+import com.resource.management.services.model.LastUpdatedTimestamp;
+import com.resource.management.services.model.LastUpdatedTimestampRepository;
+import com.resource.management.services.model.ServiceRepository;
+
+@Service
+public class LastUpdatedTimestampService
+{
+   @Autowired
+   private ServiceRepository repository;
+
+   @Autowired
+   private LastUpdatedTimestampRepository timestampRepository;
+
+
+   public ServicesListUpdatedNotification getLastUpdatedNotification( final String day )
+   {
+      LastUpdatedTimestamp timestampToday = null;
+      LastUpdatedTimestamp timestampTomorrow = null;
+      if ( "TODAY".equals( day ) )
+      {
+         timestampToday = new LastUpdatedTimestamp( "timeStampToday", Instant.now().toString() );
+         timestampTomorrow = this.timestampRepository.findById( "timeStampTomorrow" ).orElse( null );
+         this.timestampRepository.save( timestampToday );
+      }
+      else if ( "TOMORROW".equals( day ) )
+      {
+         timestampToday = this.timestampRepository.findById( "timeStampToday" ).orElse( null );
+         timestampTomorrow = new LastUpdatedTimestamp( "timeStampTomorrow", Instant.now().toString() );
+         this.timestampRepository.save( timestampTomorrow );
+      }
+      else
+      {
+         timestampToday = this.timestampRepository.findById( "timeStampToday" ).orElse( null );

```

Comment:
Use constants for the IDs and hide the find in a method in the repo, including the oreElse part.
The getTodaysTimestamp could return a nullable .
then the code would look cleaner.

Also, in my opinion a switch would look better.